usr/sbin/regionsrv-enabler-azure

```diff
@@ -97,20 +98,33 @@ utils.start_logging()
 service_name = 'guestregister'
 license_type = get_license_type()
 
-if ('BYOS' in license_type and has_license_changed(license_type) and
-    not utils.uses_rmt_as_scc_proxy()):
+if ('BYOS' in license_type and not utils.uses_rmt_as_scc_proxy()):
```

Comment:
I think it would be great to have some contextual comments in the code. Looking at it all a while after the initial implementation I think those comments would greatly help. For this condition the comment could be `# Registered to SCC`

Reply:
I'll add them

Reply:
Done

Reply:
The diff is not showing the contextual comment added per the previous comment. For this condition I suggest something along the lines of:
`# Instance has been set to BYOS via AHB but has not been registered to the update infrastructure with a registration code`